packages/broker/src/plugins/testnetMiner/TestnetMinerPlugin.ts

```diff
@@ -57,7 +58,14 @@ export class TestnetMinerPlugin extends Plugin<TestnetMinerPluginConfig> {
             this.natType = await this.getNatType()
         }
         await this.streamrClient!.subscribe(this.pluginConfig.rewardStreamId, (message: any) => {
-            this.onRewardCodeReceived(message.rewardCode)
+            if (message.rewardCode) {
+                this.onRewardCodeReceived(message.rewardCode)
+            } if (message.info) {
+                logger.info(`Info message received: ${message.info}`)
```

Comment:
Since we control the message content, maybe better to just drop the preceding text:
```suggestion
                logger.info(message.info)
```